sqlsmith: TestGenerateParse fails in CI

https://teamcity.cockroachdb.com//viewLog.html?buildId=2861471&buildTypeId=Cockroach_MergeToMaster

```
I210408 10:11:13.630947 9126 5@util/log/event_log.go:32  [n1,client=127.0.0.1:48176,hostssl,user=root] 653 ={"Timestamp":1617876673625007712,"EventType":"create_database","Statement":"CREATE DATABASE db_1","User":"root","DescriptorID":55,"DatabaseName":"db_1"}
    sqlsmith_test.go:111: RESTORE TABLE defaultdb.public.seed FROM 'http://127.0.0.1:35431/backup_3' WITH into_db = 'into_db':::STRING: at or near ":": syntax error
W210408 10:11:13.636719 10591 kv/kvserver/intentresolver/intent_resolver.go:557  [-] 654  failed to cleanup transaction intents: failed to resolve intents: node unavailable; try another peer
W210408 10:11:13.637006 8359 jobs/registry.go:729  [-] 655  canceling all adopted jobs due to stopper quiescing
W210408 10:11:13.637535 8358 sql/sqlliveness/slinstance/slinstance.go:183  [n1] 656  exiting heartbeat loop
--- FAIL: TestGenerateParse (2.18s)
```

--- a/pkg/internal/sqlsmith/bulkio.go
+++ b/pkg/internal/sqlsmith/bulkio.go
@@ -144,7 +144,7 @@ func makeRestore(s *Smither) (tree.Statement, bool) {
 		From:    []tree.StringOrPlaceholderOptList{{tree.NewStrVal(name)}},
 		AsOf:    makeAsOf(s),
 		Options: tree.RestoreOptions{
-			IntoDB: tree.NewDString("into_db"),
+			IntoDB: tree.NewStrVal("into_db"),
 		},
 	}, true
 }

